src/bootstrap.php

```diff
@@ -2,6 +2,6 @@
 
 declare(strict_types=1);
 
-if (PHP_VERSION_ID < 70300) {
+if (PHP_VERSION_ID < 70300 && !class_exists('\JsonException')) {
```

Comment:
I'd suggest to use composer classmap instead, see symfony polyfill which ensures the class is not loaded when it's not needed